docs/user/contacts.md

```diff
@@ -0,0 +1,40 @@
+# Contacts
+## [Bug Report](https://github.com/vdaas/vald/issues/new?assignees=&labels=type%2Fbug%2C+priority%2Fmedium%2C+team%2Fcore&template=bug_report.md&title=)
```

Comment:
```suggestion
## Bug Report
```
you should not add link at section title

Reply:
thank you for your review.
I will fix it soon.